### PR TITLE
Touch API

### PR DESCRIPTION
**Pull request by alexandruradovici**
The PR adds the user space API for the touch driver (https://github.com/tock/tock/pull/2060).

### CODE REVIEW

File: examples/lvgl/lvgl_driver.c [hunk 1]
```diff
@@ -2,24 +2,41 @@
 
 static lv_disp_buf_t disp_buf;
 lv_disp_drv_t disp_drv;
+lv_indev_drv_t indev_drv;
 
-void screen_lvgl_driver(lv_disp_drv_t * disp, const lv_area_t * area, lv_color_t * color_p);
+static int touchpad_x = 0, touchpad_y = 0, touchpad_status = TOUCH_STATUS_RELEASED;
+static void tocuh_event (int status, int x, int y, void *ud __attribute__ ((unused))) {
```
Comment (phil-levis) [inline examples/lvgl/lvgl_driver.c:8]:
```suggestion
static void touch_event (int status, int x, int y, void *ud __attribute__ ((unused))) {
```

**Review by phil-levis — CHANGES_REQUESTED** [review]
This looks fine, but I'd like to either wait for 2.0 to settle down or have you apply this to the 2.0 branch.

**Comment by alexandruradovici** [comment]
Are you suggesting to apply this directly to Tock 2.0 with the new API?

**Comment by hudson-ayers** [comment]
> Are you suggesting to apply this directly to Tock 2.0 using the new API?

Yes, I think so.

**Pull request closed without merging**